README.md

```diff
@@ -72,29 +72,23 @@ Afterwards, run `docker-compose up --build` to start building!
 ### Without docker
 
 - Install Java: http://openjdk.java.net/ or http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html
-- Download [Datomic here](https://my.datomic.com/downloads/free/0.9.5561) rename the 0.9.5561 to 0.9.5561.zip and unzip it into a directory.
-- Start Datomic by going to the unziped directory and run:
+- Download [Datomic here](https://my.datomic.com/downloads/free/0.9.5561), and unzip it into a directory.
+- Start Datomic by going to the unzipped directory and run:
 
 On Windows: `bin\transactor config/samples/free-transactor-template.properties`
 
-On Unix: `bin/transactor config/samples/free-transactor-template.properties`
+On Mac/Unix: `bin/transactor config/samples/free-transactor-template.properties`
 
-On macOS: [homebrew](https://brew.sh/) to do this pretty quickly:
 
-```
-brew install datomic
-brew services start datomic
-```
-
-- Install leiningen: https://leiningen.org/ into a directory. 
+- Install leiningen: https://leiningen.org/#install into a directory.
 - download the code from your git fork `git clone git@github.com:yourrepo/your.git` Use the clone url in YOUR repo.
 - cd into orcpub
 - run `lein repl`
 - run `lein figwheel` Once lein figwheel finishes, a browser will launch.
 
 You should leave all three processes running: Datomic transactor, lein repl, and lein figwheel.  
 
-*NOTE:* There is an issue using leiningen 2.8.1 causing a `ClassCastException`. Building with leiningen 2.7.1 still works
+(*NOTE:* There is an issue using leiningen 2.8.1 causing a `ClassCastException`.)
```

Comment:
Let's add 
*nix versions can use 2.9.1 (at least what I've been seeing) 
Windows ONLY works on 2.8.1  - 2.9.1 has issues with lein repl on windows.

Reply:
Just to add, I don't have a mac, can you confirm that 2.9.1 works on a mac?

Reply:
Sure, will do. I've been using 2.9.1 on my mac, but I'll double-check.

Reply:
Yeah, 2.9.1 works just fine on my mac.